package.json

```diff
@@ -3,8 +3,8 @@
     "private": true,
     "version": "0.4.1",
     "engines": {
-        "node": "12.18.3",
-        "yarn": "1.22.4"
+        "node": "16.20.2",
+        "yarn": "1.22.21"
```

Comment:
Will also have to update the `.github/workflows/nodejs.yaml` to use the new versions of node and yarn

Line 8
```yaml
node-version: [16.20.2]
```

Line 15
```yaml
- run: npm install -g yarn@1.22.21
```